components/home/OfficialChannelList.vue

```diff
@@ -70,6 +101,47 @@ export default {
   components: {
     ImageGrid,
   },
+  data() {
+    return {
+      newsletters: [
+        {
+          title: 'LikeCoin Annual Report 2019',
+          pubDate: '2020-01-01 10:43:17',
```

Comment:
Better add timezone also?

Reply:
Actually I don't quite understand, since we just need the date string, why not just put the date string here, so we don't need to care about local time conversion and prevent inconsistency?